ui/html/ui-tools.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Lamassu Machine Dev Tools</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <style>
+    body {
+      background-color: #82dda3;
+      font-family: 'MuseoSans', sans-serif;
+    }
+    main {
+      max-width: 42rem;
+      margin: 3rem auto;
+    }
+    button, input {
+      background-color: #fff;
+      border: none;
+      padding: 0.5rem 1rem;
+      border-radius: 0.5rem;
+      font-size: 1.2rem;
+    }
+    button { cursor: pointer; }
+    button:hover { background-color: #f0f0f0; }
+    button:focus, input:focus { outline: 2px solid #3d65ec; }
+    .container {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 1rem;
+      background-color: #66ba85;
+      padding: 2rem;
+      border-radius: 0.5rem;
+    }
+    #custom-bill-input { width: 8rem; }
+    input { flex-grow: 1; }
+    .in { margin-left: auto; }
+    .address-row {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: space-between;
+      width: 100%;
+    }
+    .address-row > input { margin-left: 1rem; margin-right: 1rem; }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Dev Tools for lamassu-machine</h1>
+    <div id="pairing-token">
+      <h2>Pairing Token</h2>
+      <div class='container'>
+        <input type='text' id='token-input' placeholder='Your pairing token...' oninput='validity.valid||(value="");'>
+        <button id='token-submit' title='Insert' onclick='insertPairingToken()'>▸</button>
+      </div>
+    </div>
+    <div id="wallet-addresses">
+      <h2>Wallet Addresses</h2>
+      <div class='container'>
+        <div class="address-row">
+          <h3>BTC</h3>
+          <input type='text' id='btc-address-input' placeholder='Your BTC address...' oninput='validity.valid||(value="");'>
```

Comment:
No need for validity check.

```suggestion
          <input type='text' id='btc-address-input' placeholder='Your BTC address...'>
```